src/docfx/build/xref/XrefResolver.cs

```diff
@@ -145,7 +145,7 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                 {
```

Comment:
Apply this patch only when `outputUrlType == Docs`?

Reply:
Why should add this logic in this time, I mean the original logic is also applied to all the `outputUrlType`. and in our spec, we already consume the live branch

---
In reply to: [477133335](https://github.com/dotnet/docfx/pull/6450#discussion_r477133335) [](ancestors = 477133335)

Reply:
doesn't need to be in this PR